Sources/FeaturevisorTypes/Types.swift

```diff
@@ -0,0 +1,588 @@
+import Foundation
```

Comment:
we renamed it from `types.swift` to `Types.swift`.

for some reasons, we cannot see the full diff because of that. is it possible to keep it as `types.swift` at least on this PR?

also, when it's only the casing that gets changed in a file, the combo of `macOS + Git` will be a pain for others when they will pull latest commits 😬

Reply:
Done, reverted to types. I will update the file names to start from the upper case under separate PR. Generally. in Swift the files starts from uppercase